libs/sdk-ui-kit/src/Header/HeaderHelp.tsx

```diff
@@ -77,15 +77,34 @@ export const CoreHeaderHelp: React.FC<IHeaderHelpProps> = ({
         onMenuItemClick(...args);
     };
 
+    /**
+     * Menu dropdown content is long enough to make it max-width (230px),
+     * so it should just switch alignment to bottom right corner of the Help button.
+     */
+    const getHelpDropdownAlignPoints = (): IAlignPoint[] => {
+        const defaultAlignPoints = [
+            {
+                align: "br tr",
+            },
+        ];
+
+        if (!helpMenuDropdownAlignPoints || helpMenuDropdownAlignPoints === "br tr") {
+            return defaultAlignPoints;
+        }
+
+        return [
```

Comment:
How about if the `helpMenuDropdownAlignPoints` different `br tr` value? I think we should change like this:
```javascript
return [{
    align: helpMenuDropdownAlignPoints || "br tr",
}, {
   align: "bl tl"
}];
```

Reply:
`helpMenuDropdownAlignPoints` has two values: https://github.com/gooddata/gooddata-ui-sdk/blob/7be2a518319a02223cc2bbd78baab4cd6d7dfea7/libs/sdk-ui-kit/src/typings/positioning.ts#L53.

The logic should be:
```
[{
    align:="bl tl",
}, {
   align: "br tr"
}]
```
after rendering. "bl tl" have priority than "br tr"